mono/utils/winarm64.asm

```diff
@@ -0,0 +1,48 @@
+	AREA	|.text|,CODE
+
+	EXPORT	|mono_context_get_current|
+
+|mono_context_get_current|	PROC
+	mov	x16, x0
```

Comment:
Nit, a little bit of comments here might help

Reply:
![image](https://user-images.githubusercontent.com/56306816/207078830-1432b41a-a4be-4dca-9b2c-750058612000.png)

Reply:
![image](https://user-images.githubusercontent.com/56306816/207079232-2a5bba4e-2936-48df-9e09-5ff41d447901.png)

Reply:
The way this was written:

1. gcc/clang macro that used inline assembly was compiled for Android with Clang;
3. The compiled object file was disassembled;
4. The resulting assembly was copy pasted into this file.